### PR TITLE
V abt/meeting app

### PR DESCRIPTION
**Pull request by AbhijeetThakur-msft**
HR meeting app

### CODE REVIEW

File: samples/meeting-app/MeetingApp/MeetingApp/ClientApp/src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/MeetingApp/MeetingApp/ClientApp/src/reportWebVitals.js:8]: using if needed for supported metrics

File: samples/meeting-app/MeetingApp/MeetingApp/ClientApp/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/MeetingApp/MeetingApp/ClientApp/public/manifest.json:19]: it has react specific meta data

File: samples/meeting-app/MeetingApp/MeetingApp/ClientApp/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/MeetingApp/MeetingApp/ClientApp/public/index.html:26]: it's in public file, just some information, can remove if needed

File: README.md [hunk 4]
```diff
@@ -228,6 +229,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 [meetings-stage-view#cs]:samples/meetings-stage-view/csharp
 [meetings-events#cs]:samples/meetings-events/csharp
 [meetings-events#js]:samples/meetings-events/nodejs
+[meeting-app#cs]:samples/meeting-app/csharp
```
Comment (Wajeed-msft) [inline README.md:232]: @AbhijeetThakur-msft - can we rename this to "meetings-recruitment-app" ? Please make same change to node solution as well.

File: samples/meeting-app/csharp/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
+﻿## MeetingApp (HR recruitment) Sample
+
+This sample illustrates a meeting experience for HR recruitment.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple bot that accepts input from the user and echoes it back.
```
Comment (Wajeed-msft) [inline samples/meeting-app/csharp/README.md:5]: Could you please update the description?

File: samples/meeting-app/csharp/MeetingApp/MeetingApp.csproj [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+  <ItemGroup>
+    <PackageReference Include="AdaptiveCards" Version="2.7.2" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.10.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.18" />
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.18" />
+    <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.14.0" />
+    <PackageReference Include="Microsoft.Graph" Version="4.5.0" />
```
Comment (Wajeed-msft) [inline samples/meeting-app/csharp/MeetingApp/MeetingApp.csproj:17]: Check if we are actually using Graph SDK. If not let's remove it.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine for me, approving it.

File: README.md (hunk 4 above)
Comment (AbhijeetThakur-msft) [reply README.md]: changed

File: samples/meeting-app/csharp/MeetingApp/MeetingApp.csproj (hunk 6 above)
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/csharp/MeetingApp/MeetingApp.csproj]: removed, not required

File: samples/meeting-app/csharp/README.md (hunk 5 above)
Comment (AbhijeetThakur-msft) [reply samples/meeting-app/csharp/README.md]: changed, please check.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**